.github/workflows/test.yml

```diff
@@ -35,13 +35,19 @@ jobs:
           pip install cython
           pip install -e .
           pip install coverage
-      - name: Log complete list of packages
-        run: |
-          pip freeze
+      - run: pip freeze
+      - run: pip list
       - name: Check licenses
         run: |
           pip install pip-licenses
           if pip-licenses | grep -E -v 'Artistic License|LGPL|Public Domain' | grep GNU; then echo 'Please avoid introducing *GPL dependencies'; false; fi
+      - uses: tj-actions/changed-files@v36
```

Comment:
`tj-actions/changed-files@v36` is giving us a node 16 warning (see https://github.com/EveryVoiceTTS/EveryVoice/actions/runs/10581973532)
any reason not to use the current `@v45`?

Reply:
I'll update to v45

Reply:
I'll also try out a not properly blacked file.  I guess I'll have to use nano because my ide auto formats.